extensions/ql-vscode/src/vscode-tests/cli-integration/run-remote-query.test.ts

```diff
@@ -68,7 +68,9 @@ describe('Remote queries', function() {
   it('should run a remote query that is part of a qlpack', async () => {
     const fileUri = getFile('data-remote-qlpack/in-pack.ql');
 
-    const queryPackRootDir = (await runRemoteQuery(cli, credentials, fileUri, true, progress, token))!;
+    const querySubmissionResult = await runRemoteQuery(cli, credentials, fileUri, true, progress, token);
+    expect(querySubmissionResult).to.be.ok;
+    const queryPackRootDir = querySubmissionResult!.queryDirPath!;
```

Comment:
Not really a review comment - just a silly question... Why do we need the exclamation marks `!`s here? Is it just to convince the TypeScript compiler that these definitely exist?

Reply:
_Bang_ on. See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator